docs/adding-builtins.md

```diff
@@ -1,53 +0,0 @@
-# Adding Built-in Functions
-
```

Comment:
Is the process much simpler now?  Just drop in a new Go function and register it?